src/components/action-popover/action-popover.stories.mdx

```diff
@@ -420,13 +444,13 @@ ActionPopoverMenu. Expand the code example below for how to use this.
             <FlatTableCell>
               <ActionPopover
                 renderButton={({
-                  "data-element": dataElement,
+                  "data-component": dataComponent,
                   ariaAttributes,
                 }) => (
                   <Link
                     onClick={() => {}}
                     tabbable={false}
-                    data-element={dataElement}
+                    data-component={dataComponent}
```

Comment:
I don't think it's currently possible to override the `data-component` value of Link

Reply:
It looks like for the Link component actually neither of these two work to focus the button when ActionPopover is closed, so it looks like we simply had a bad example to show the custom button functionality

Reply:
Yeah I agree, I think we can fix this in a separate ticket if you'd prefer